renderlib/CCamera.h

```diff
@@ -352,6 +352,7 @@ class CCamera
     m_Dirty = true;
     m_Projection = PERSPECTIVE;
     m_OrthoScale = DEF_ORTHO_SCALE;
+    m_AreaPixel = m_Film.m_Resolution.GetAspectRatio() / (m_Focus.m_FocalDistance * m_Focus.m_FocalDistance);
```

Comment:
What does `m_AreaPixel` represent? The area of a pixel at the focal distance?

Reply:
yeah this is just a throw-in for a lint warning I was seeing about uninitialized data.  I don't think the value is even used anywhere currently.  could be used for depth-of-field blurring or adaptive sampling bla bla bla.